querybook/server/logic/datadoc_collab.py

```diff
@@ -177,7 +177,7 @@ def update_data_cell(cell_id, fields, sid="", session=None):
     assert_can_write(data_doc.id, session=session)
     verify_environment_permission([data_doc.environment_id])
     session.commit()
-
+    logic.update_es_query_cell_by_id(data_cell.id)
```

Comment:
hm i guess the logic is a bit convoluted now, lets do `assert_can_write` and `verify_environment_permission` first before update_data_cell and have the es update logic there